static/bidder-info/resetdigital.yaml

```diff
@@ -0,0 +1,19 @@
+    endpoint: http://b-us-east14.resetdigital.co:9001
+    meta-info:
+      maintainer-email: biddersupport@resetdigital.co
```

Comment:
@bruno-siira  Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread

Reply:
@bruno-siira  email bounced back. `biddersupport@resetdigital.co` is not reachable

Reply:
Email was having issue, can resent email to verify?

Reply:
@bruno-siira Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread

Reply:
We had a problem to get mails from external sources. We solved it now, can you retest it?

Reply:
received response from ResetDigital